### ISSUE TITLE
If I don't use stratify=labels during train_test_split this error appears

### ISSUE THREAD

**Issue by Aakanksha3010** [rohanaggarwal45/Face-mask-detection-in-Colab#1]
![image](https://user-images.githubusercontent.com/69424461/108403703-0230e200-7245-11eb-89de-7fbe0e5382b0.png)
